### PR TITLE
Adds config.yml for Issue Templates

### PR DESCRIPTION
**Pull request by dmikusa-pivotal**
One more try, https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser says that you need a config.yml to configure the issue template chooser. Let's see if adding that help.

### CODE REVIEW

File: ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```
Comment (fg-j) [inline ISSUE_TEMPLATE/config.yml:1]: Do you think it's a problem to never allow blank issues?

Comment (fg-j) [reply ISSUE_TEMPLATE/config.yml:1]: Eh. I'm fine with it for now and we can tweak if this gets things working.

**Review by fg-j — APPROVED** [review]

**Comment by fg-j** [comment]
Sadly I still don't see this working :(

**Comment by dmikusa-pivotal** [comment]
Same. I'm out of ideas. I would reach out to Github Support, but I'm going to be OOO for a week. @fg-j or @TisVictress to you want to open an issue with GH support and see if they can point us to the problem? Hopefully it's not way off.

**Comment by fg-j** [comment]
I just filed a ticket with Github Support.

**Pull request merged**